### PR TITLE
add Setup for docker

### PR DESCRIPTION
**Pull request by davidbradway**

### CODE REVIEW

File: provision.sh [hunk 1]
```diff
@@ -36,3 +42,16 @@ apt-get -y install vim
...
 apt-get -y install screen
 
+### Install Docker
+# https://docs.docker.com/engine/installation/linux/docker-ce/ubuntu/#install-using-the-convenience-script
+curl -fsSL get.docker.com -o get-docker.sh
+sh get-docker.sh
+rm get-docker.sh
+
+### Setup Docker without sudo
+usermod -aG docker vcm
```
Comment (suyashkumar) [inline provision.sh:52]: should we have vcm here have the same check as above on line 27?

**Review by suyashkumar — APPROVED** [review]
LGTM!

File: provision.sh (hunk 1 above)
Comment (davidbradway) [reply provision.sh:52]: Good catch. Please edit and merge.

**Comment by davidbradway** [comment]
see other PR

**Pull request closed without merging**